ext/hal/nordic/Kconfig

```diff
@@ -26,6 +26,9 @@ config NRFX_PWM
 config NRFX_SAADC
 	bool
 
+config NRFX_QDEC
```

Comment:
Keep the alphabetical order of the peripherals, please.

Reply:
Fixed